### PR TITLE
185 enable linters

### PR DESCRIPTION
**Pull request by DarjaLeonova**

### CODE REVIEW

File: cmd/client/client_integration_test.go [hunk 1]
```diff
@@ -96,7 +96,10 @@ func mustGetFreePort() string {
 	defer l.Close()
 
 	// Get the port number from the address that the Listener is listening on.
-	addr := l.Addr().(*net.TCPAddr)
+	addr, ok := l.Addr().(*net.TCPAddr)
```
Comment (DarjaLeonova) [inline cmd/client/client_integration_test.go:99]: Changed cause of forcetypeassert linter
https://github.com/gostaticanalysis/forcetypeassert

File: cmd/client/client_integration_test.go [hunk 2]
```diff
@@ -305,14 +308,14 @@ func Test_TranslationFileUpload_CLI(t *testing.T) {
 
 		require.NoError(t, err)
 
-		_, err = file.Write([]byte(`
+		_, err = file.WriteString(`
```
Comment (DarjaLeonova) [inline cmd/client/client_integration_test.go:311]: Changed cause of mirror linter
https://github.com/butuzov/mirror

File: pkg/model/message.go [hunk 3]
```diff
@@ -11,9 +11,9 @@ import (
 )
 
 type Translation struct {
-	Language language.Tag
-	Messages []Message
-	Original bool
+	Language language.Tag `json:"language"`
```
Comment (DarjaLeonova) [inline pkg/model/message.go:14]: Changed cause of musttag linter.
https://github.com/go-simpler/musttag

File: pkg/model/services.go [hunk 4]
```diff
@@ -3,6 +3,6 @@ package model
 import "github.com/google/uuid"
 
 type Service struct {
-	Name string
-	ID   uuid.UUID
+	Name string    `json:"name"`
```
Comment (DarjaLeonova) [inline pkg/model/services.go:6]: Changed cause of musttag linter.
https://github.com/go-simpler/musttag

File: .golangci.yml [hunk 5]
```diff
@@ -5,72 +5,43 @@ run:
   skip-dirs:
     - pkg/pb
 linters:
-  enable-all: true
+  enable:
```
Comment (janishorsts) [inline .golangci.yml:8]: Why did you change `enable-all`? It should stay as-is and we just disable the linters we are not interested in.

File: .golangci.yml [hunk 6]
```diff
@@ -5,72 +5,43 @@ run:
...
+   - forcetypeassert
+   - mirror
+   - musttag
   disable:
    - asasalint
-   - cyclop
+   - gci
```
Comment (janishorsts) [inline .golangci.yml:14]: please sort the linters alphabetically for consistency
